timescaledb/how-to-guides/continuous-aggregates/compression-on-continuous-aggregates.md

```diff
@@ -0,0 +1,63 @@
+# Compression on continuous aggregates
+Continuous aggregates are often used to store downsampled historical data.
+The historical data is almost never modified or recomputed and is only used 
+for serving analytic queries. For this use case, it is often beneficial to 
+store the materialized data in compressed form to save on storage costs. 
+You can get these cost savings by enabling compression on continuous 
+aggregates.
+
+Currently, TimescaleDB does not support refreshing compressed regions of a 
+continuous aggregate. To do this, you have to manually decompress 
+the compressed chunk and then execute a `refresh_continuous_aggregate` call.
+
+## Enable compression on continuous aggregates
+You can enable and disable compression on continuous aggregated by setting
+`compress` parameter when you alter the view.
+
+<procedure>
+
+### Enabling and disabling compression on continuous aggregates
+1.  For an existing continuous aggregate, at the `psql` prompt, enable
+ compression:
+    ```sql
+    ALTER MATERIALIZED VIEW cagg_name set (timescaledb.compress = true);
+    ```
+1.  Disable compression:
+    ```sql
+    ALTER MATERIALIZED VIEW cagg_name set (timescaledb.compress = false);
+    ```
+</procedure>
+The decompress command fails if there are compressed chunks associated with the 
+continuous aggregate. In this case, you need to decompress the chunks, and then 
+drop any compression policy on the continuous aggregate, before you disable 
+compression. For more detailed information, see the
+[decompress chunks] [decompress-chunks] section:
+```sql
+SELECT decompress_chunk(c, true) FROM show_chunks('cagg_name') c;
+ 
+
+## Compression policies on continuous aggregates
+Before  setting up a compression policy on a continuous aggregate, you should
```

Comment:
should we specify that you _have_ to set up a refresh policy before you set up a compression policy?

Reply:
"must" ?

Reply:
Currently, you really "have to" based on yesterday's conversation.